docs/testing/steps.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Steps
+type: docs
+menu:
+  docs:
+    parent: 'Testing'
+weight: 3
+---
+
+# Steps
+
+* [Format](#format)
+* [Creating objects](#creating-objects)
+* [Updating objects](#updating-objects)
+* [Deleting objects](#deleting-objects)
+* [Kubectl commands](#kubectl-commands)
+
+Each test case is broken down into test steps. Test steps within a test case are run sequentially: if any of the test steps fail, the entire test case is considered failed.
+
+A test step can create, update, and delete objects as well as run any kubectl command.
+
+# Format
+
+A test step can include many YAML files and each YAML file can contain many Kubernetes objects. In a test case's directory, each file that begins with the same index is considered a part of the same test step.
+
+E.g., in a test case directory:
+
+```
+tests/e2e/example/00-pod.yaml
```

Comment:
and inside one step, the files are applied in any particular order, right?

Reply:
Yeah, I'll write that they should logically be considered simultaneous.